### PR TITLE
visualization server wants kubernetes serivce account too

### PR DESCRIPTION
**Pull request by jingzhang36**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2822)

### CODE REVIEW

File: manifests/kustomize/gcp-workload-identity-setup.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ SYSTEM_GSA=${SYSTEM_GSA:-$CLUSTER_NAME-kfp-system}
 USER_GSA=${USER_GSA:-$CLUSTER_NAME-kfp-user}
 
 # Kubernetes Service Account (KSA)
-SYSTEM_KSA=(ml-pipeline-ui)
```
Comment (Bobgy) [inline manifests/kustomize/gcp-workload-identity-setup.sh]: Visualization server is currently using default service account.
You need to add a specific service account for it first.

You can follow examples here:
https://github.com/kubeflow/pipelines/blob/10a4f55e3ae5ed0e47466ec3bd4f0d56b9333cec/manifests/kustomize/base/pipeline/ml-pipeline-viewer-crd-deployment.yaml#L27
https://github.com/kubeflow/pipelines/blob/10a4f55e3ae5ed0e47466ec3bd4f0d56b9333cec/manifests/kustomize/base/pipeline/ml-pipeline-viewer-crd-sa.yaml#L4

Comment (Bobgy) [reply manifests/kustomize/gcp-workload-identity-setup.sh]: Reminder, also update the same change in hosted manifests: https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml#L444

Comment (jingzhang36) [reply manifests/kustomize/gcp-workload-identity-setup.sh]: done

File: manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml [hunk 2]
```diff
@@ -608,6 +608,7 @@ spec:
           name: ml-pipeline-visualizationserver
           ports:
             - containerPort: 8888
+      serviceAccountName: ml-pipeline-visualizationserver
```
Comment (Bobgy) [inline manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml]: You also need to add the service account yaml for hosted.

Comment (jingzhang36) [reply manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml]: done

**Comment by Bobgy** [comment]
@rmgogogo Can you approve too?

**Pull request merged**